frontend/src/components/widgets/floating-menus/FloatingMenu.vue

```diff
@@ -240,66 +262,29 @@ export default defineComponent({
 			}
 		}
 	},
+	unmounted() {
+		window.removeEventListener("mousedown", this.mouseDownHandler, true);
+		window.removeEventListener("mouseup", this.mouseUpHandler, true);
+	},
 	methods: {
 		setOpen() {
-			this.open = true;
+			this.$emit("update:isOpen", true);
 		},
 		setClosed() {
-			this.open = false;
-		},
-		isOpen(): boolean {
-			return this.open;
-		},
-		getWidth(callback: (width: number) => void) {
-			this.$nextTick(() => {
-				const floatingMenuContent = this.$refs.floatingMenuContent as HTMLElement;
-				const width = floatingMenuContent.clientWidth;
-
-				callback(width);
-			});
+			this.$emit("update:isOpen", false);
 		},
-		disableMinWidth(callback: (minWidth: string) => void) {
-			this.$nextTick(() => {
-				const floatingMenuContent = this.$refs.floatingMenuContent as HTMLElement;
-				const initialMinWidth = floatingMenuContent.style.minWidth;
-				floatingMenuContent.style.minWidth = "0";
-
-				callback(initialMinWidth);
-			});
-		},
-		enableMinWidth(minWidth: string) {
+		// Measures the width this menu should have, absent minWidth constraints.
+		async measureNaturalWidth(): Promise<number | undefined> {
 			const floatingMenuContent = this.$refs.floatingMenuContent as HTMLElement;
-			floatingMenuContent.style.minWidth = minWidth;
-		},
-		mouseMoveHandler(e: MouseEvent) {
-			const MOUSE_STRAY_DISTANCE = 100;
```

Comment:
Mouse stray behavior was lost.

Reply:
Oh, we chatted about that in the Discord. I thought the conclusion was that we didn't need that behavior because it doesn't match what other menu systems (eg. VS Code) do

Reply:
Ah, I'm sorry if that was unclear. I believe we were talking about how submenus persist after un-hovering the entry which spawns the submenu (VS Code works by keeping it around for about a second, while we make it disappear instantly). The stray behavior matches how it works in Blender and it's intended behavior across all floating menus, like popovers, dropdown menus, and the color picker.